packages/react/src/components/DataTable/DataTable.js

```diff
@@ -373,6 +373,8 @@ export default class DataTable extends React.Component {
       useStaticWidth,
       shouldShowBorder,
       stickyHeader,
+      'aria-labeledby': `${this.getTablePrefix()}__table-container__title`,
```

Comment:
```suggestion
      'aria-labelledby': `${this.getTablePrefix()}__table-container__title`,
```
For some reason, ARIA uses the British English "labelled" not the American English "labeled" 🤷‍♀

Reply:
Good catch - Thanks @abbeyhrt!